invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/index.js

```diff
@@ -51,11 +51,32 @@ if (recordCitationAppDiv) {
 
 $(".ui.accordion").accordion({
   selector: {
-    trigger: '.title .dropdown'
+    trigger: '.title'
   }
 });
 
-$("#record-doi-badge").click(function () {
+$('.ui.accordion .title')
+  .on('keydown', function(event) {
+    if($(event.target).is('.title') && event.key === "Enter") {
+      let classList = Array.from(event.target.classList);
+
+      if(classList.indexOf('active') > -1) {
+        $(event.target).accordion('close');
+      }
+      else {
+        $(event.target).accordion('open');
+      }
+    }
+  });
+
+$(".ui.accordion.affiliations-accordion").accordion({
+  selector: {
+    trigger: '.title .affiliations-button'
+  }
```

Comment:
Check if this conflicts with https://github.com/inveniosoftware/invenio-app-rdm/pull/1151